project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementService.java

```diff
@@ -665,6 +585,23 @@ public CompletableFuture<List<Result<MeasurementId, ErrorCode>>> updateAll(
       List<MeasurementMetadata> measurementMetadataList, ProjectId projectId) {
     List<Result<MeasurementId, ErrorCode>> results;
 
+    if (!measurementMetadataList.isEmpty() && measurementMetadataList.get(
```

Comment:
as this method became more compley, it would be helpful to add some javadoc to the method